From d93abf7bc1af97c42d6ce0baf1e89071b62da6d3 Mon Sep 17 00:00:00 2001
From: Wendy Liang <wendy.liang@xilinx.com>
Date: Mon, 7 Aug 2017 23:49:45 -0700
Subject: [PATCH 269/532] remoteproc: zynqmp_r5: Use () around macro argument

commit 1ab9bb409b8e5f6c7973487ce6b6d979a8f43190 from
https://github.com/Xilinx/linux-xlnx.git

Macro argument is better put in '()' as '(ARG)' in macro definition to
avoid precedence issues.

Signed-off-by: Wendy Liang <jliang@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/remoteproc/zynqmp_r5_remoteproc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/remoteproc/zynqmp_r5_remoteproc.c b/drivers/remoteproc/zynqmp_r5_remoteproc.c
index 3f8edb0..4e28437 100644
--- a/drivers/remoteproc/zynqmp_r5_remoteproc.c
+++ b/drivers/remoteproc/zynqmp_r5_remoteproc.c
@@ -70,7 +70,7 @@
 
 /* RPU IPI mask */
 #define RPU_IPI_INIT_MASK	0x00000100
-#define RPU_IPI_MASK(n)		(RPU_IPI_INIT_MASK << n)
+#define RPU_IPI_MASK(n)		(RPU_IPI_INIT_MASK << (n))
 #define RPU_0_IPI_MASK		RPU_IPI_MASK(0)
 #define RPU_1_IPI_MASK		RPU_IPI_MASK(1)
 
-- 
2.7.4

